docs/docs/guides/integrate/login-ui/typescript-repo.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Typescript Repository
+sidebar_label: Typescript Repository
+---
+
+To replace our current golang built login UI and showcase the use of our new resource and OIDC APIs, we've created the [Typescript Repository](https://github.com/zitadel/typescript).
```

Comment:
Maybe we should mention the session API here, Do we have a new OIDC API?:

```suggestion
To replace our current golang built login UI and showcase the use of our new resource, session and OIDC APIs, we've created the [Typescript Repository](https://github.com/zitadel/typescript).
```

Reply:
> Maybe we should mention the session API here, Do we have a new OIDC API?:

I'd address this better with a drawing of the architecture of the new login.